app/src/main/res/menu/main.xml

```diff
@@ -11,6 +11,12 @@
         app:actionViewClass="android.support.v7.widget.SearchView"
         app:showAsAction="always" />
     <item
+        android:id="@+id/action_folder"
```

Comment:
This can totally be added, but I would like to ask you to make that optionally toggleabel, so its still showing the 2 fabs by default, and switching to the toolbar option way when enabled.

In general fab is just for primary action right, but got used to it in this app and quite like it in this case.

Reply:
ok cool , don't think putting int setting will be better , let just leave like that

Reply:
k